### ISSUE TITLE
Possibility to invite a registered user to a space by email

### ISSUE THREAD

**Issue by funkycram** [humhub/humhub#5005]
### Is your feature request related to a problem? Please describe.

If you invite to a space persons by email but some of them have already an account, you get the error message "{email} is already registered!" (see https://github.com/humhub/humhub/blob/2f8623264ca90e4d57ba7fa0a1479276f6c7d1f1/protected/humhub/modules/space/models/forms/InviteForm.php#L240).

This becomes problematic if you invite many persons (e.g. 100 persons) because you need to find one by one which email is associated to an account an remove their email from the invitation form

### Describe the solution you'd like

Instead of throwing an error, add membership to the space to persons not registered and to users whose email is in the list.

### Describe alternatives you've considered

Instead of throwing an error,  add membership to the space to persons not registered and show the list users not added because they are already registered

**Comment by yurabakhtin** [comment]
@luke- PR https://github.com/humhub/humhub/pull/5164.

**Comment by luke-** [comment]
@yurabakhtin Thanks!

**Comment by funkycram** [comment]
Great! Thanks @yurabakhtin !

**Comment by funkycram** [comment]
It seems that PR #5164 has not been integrated into Humhub 1.9 (it should have been in beta 2 ([see here](https://github.com/humhub/humhub/blob/714a136ce54fbdf04bde542139b9a0bf0f075a2d/CHANGELOG_DEV.md)) but unfortunately it has not ([see releases](https://github.com/humhub/humhub/releases)) and it is not in the [Develop changelog](https://github.com/humhub/humhub/blob/develop/CHANGELOG_DEV.md) for the next release.

If it was, https://github.com/humhub/humhub/blob/master/protected/humhub/modules/space/models/forms/InviteForm.php should have the `addUserToInviteList` method.

**Comment by yurabakhtin** [comment]
@funkycram Yes, you are right it will be included in next release probably in `1.10.0-beta.1`, because current version of the `develop` branch has such version.
The `develop` branch had version `1.9.0-beta.2` at that moment when I developed the PR https://github.com/humhub/humhub/pull/5164, but it is still not merged into `develop`.

@luke- Should we move the doc https://github.com/humhub/humhub/pull/5164/files#diff-df55e56bfe234313eade0c101b42fc90f37782988f200fc61c311226b9bc4c4bR13 into the proper version part? I guess firstly we should merged `develop` branch into the PR and only after that to do the doc update. Let me know if I should do here something.

**Comment by luke-** [comment]
@yurabakhtin Oh yes, would be great if you move the changelog entry to the proper version.

**Comment by yurabakhtin** [comment]
@luke- Ok, I have synced the PR with `develop` branch here https://github.com/humhub/humhub/pull/5164/commits/0500f5d44bb0da23f737e33e2d35f646d722e043.
